Modules/Messages/src/Messages.Infrastructure/Persistence/Database/Repository/MessagesRepository.cs

```diff
@@ -72,10 +67,10 @@ public Task<DbPaginationResult<Message>> FindMessagesOfIdentity(IdentityAddress
         return query.OrderAndPaginate(d => d.CreatedAt, request.PaginationFilter);
     }
 
-    public EntityEntry<Message> Update(Message message)
+    public Message Update(Message message)
     {
         var m = _messages.Update(message);
         _dbContext.SaveChanges();
-        return m;
+        return m.Entity;
```

Comment:
No need to return the Message. Whoever passes it to the method already has it - otherwise they couldn't have passed it. ;) 
It only causes confusion, because you would wonder if the returned Message is different than the one that was passed.